resources/views/partials/navbar/menu-item-dropdown-user-menu.blade.php

```diff
@@ -1,6 +1,10 @@
 @php( $logout_url = View::getSection('logout_url') ?? config('adminlte.logout_url', 'logout') )
 @php( $profile_url = View::getSection('profile_url') ?? config('adminlte.profile_url', 'logout') )
 
+@if config('adminlte.usermenu_profile_image')
```

Comment:
I believe this line should be `@if config('adminlte.usermenu_profile_url')` instead

Reply:
@Shidersz you are right, I will fix that 👍